src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.NonAndroid.cs

```diff
@@ -379,6 +421,22 @@ private static bool TryLoadTzFile(string tzFilePath, [NotNullWhen(true)] ref byt
             return false;
         }
 
+#if TARGET_WASI || TARGET_BROWSER
+        private static bool TryLoadEmbeddedTzFile(string name, [NotNullWhen(true)] ref byte[]? rawData)
```

Comment:
I would suggest using `out` instead of `ref`. It would better follow the `Try*` pattern. And the effect will be the same, except that the method will be forced to set a value.